### PR TITLE
D8CORE-5741 | reorder opportunity node display

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- reorder opportunity node display

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Spin up gitpod
3. Navigate to the opportunity node display `/admin/structure/types/manage/stanford_opportunity/display`
image

5. Verify that the grouping matches the figma mock
6. Review code

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? **YES** :tada:

# Associated Issues and/or People
- D8CORE-5741

### CODE REVIEW

File: config/sync/core.entity_view_display.node.stanford_opportunity.default.yml [hunk 1]
```diff
@@ -155,14 +197,14 @@ content:
...
     region: content
   su_opp_prerequisites:
     type: text_default
     label: above
     settings: {  }
     third_party_settings: {  }
-    weight: 118
+    weight: 8
     region: content
   su_opp_source:
```
Comment (rebeccahongsf) [inline config/sync/core.entity_view_display.node.stanford_opportunity.default.yml:209]: I'm unsure about the ordering of the External Link `su_opp_source` and Sponsor `su_opp_sponsor` 🤔

Comment (buttonwillowsix) [reply config/sync/core.entity_view_display.node.stanford_opportunity.default.yml:209]: @rebeccahongsf The External link should not be displayed (please hide on the display. Only used for Rabbit hole.
I think the Sponsor can go below the page title.

**Review by buttonwillowsix — CHANGES_REQUESTED** [review]
I left some comments on changes.
I think unless need, we don't need to show the labels, either.

**Comment by rebeccahongsf** [comment]
@buttonwillowsix I'm going to do a quick rebuild off the latest from your branch since I believe it would be quicker than resolve the merge conflicts 😃

**Pull request closed without merging**